### ISSUE TITLE
Panic on startup

### ISSUE THREAD

**Issue by ButteryGuitars** [paritytech/parity-ethereum#9624]
_Before filing a new issue, please **provide the following information**._

- **Parity Ethereum version**: 2.1.1
- **Operating system**: Windows
- **Installation**: one-line installer
- **Fully synchronized**: yes
- **Network**: ethereum
- **Restarted**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

Have a fully synchronized database that was pulled down with v2.0.6. Attempting startup v2.1.1 and it panics right out of the gate. Here is debug output,

C:\Program Files\Parity Technologies\Parity>parity.exe -l debug --log-file err.txt
Loading config file from C:\Users\user\AppData\Roaming\Parity\Ethereum\config.toml

====================

stack backtrace:
   0:        0x140da0623 - hid_error
   1:        0x140da029e - hid_error
   2:        0x140da031d - hid_error
   3:        0x140d9f6a3 - hid_error
   4:        0x140d9f41d - hid_error
   5:        0x140df8a62 - hid_error
   6:        0x140df85c4 - hid_error
   7:        0x140df84a9 - hid_error
   8:        0x140e1339c - hid_error
   9:        0x13f948e5c - <no info>
  10:        0x13f7a60bd - <no info>
  11:        0x13f8966e3 - <no info>
  12:        0x13f899834 - <no info>
  13:        0x13f701bf6 - <no info>
  14:        0x140df8407 - hid_error
  15:        0x140e0f062 - hid_error
  16:        0x140df26c3 - hid_error
  17:        0x13f89a73a - <no info>
  18:        0x14113b41c - hid_write
  19:         0x779759cd - BaseThreadInitThunk

Thread 'main' panicked at 'Logger is initialized only once; qed: "Cannot write to log file given: err.txt"', libcore\result.rs:945

This is a bug. Please report it at:

    https://github.com/paritytech/parity-ethereum/issues/new

**Comment by joshua-mir** [comment]
User is also experiencing another crash on startup that happens *before any logs can be collected* (clearly hard to collect these logs)

**Comment by 5chdn** [comment]
Can you try

    parity.exe -l debug --log-file C:\Users\user\Desktop\err.txt

**Comment by ButteryGuitars** [comment]
Just gave it a shot. Here is crash
[err.txt](https://github.com/paritytech/parity-ethereum/files/2424273/err.txt)

**Comment by 5chdn** [comment]
What crash?

**Comment by ButteryGuitars** [comment]
![image](https://user-images.githubusercontent.com/43388326/46211935-66604d80-c2fa-11e8-9b16-93e88f0f3539.png)
 
that is the error I get when I launch parity using any options with or without a config.toml in place. is there another option I can use that would give more info to help?

**Comment by ButteryGuitars** [comment]
Here's the extended details of that crash popup

Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	parity.exe
  Application Version:	0.0.0.0
  Application Timestamp:	5ba2a5ef
  Fault Module Name:	KERNELBASE.dll
  Fault Module Version:	6.1.7601.23915
  Fault Module Timestamp:	59b94f2a
  Exception Code:	c0000005
  Exception Offset:	000000000000b2b2
  OS Version:	6.1.7601.2.1.0.256.4
  Locale ID:	1033
  Additional Information 1:	bb89
  Additional Information 2:	bb89233b57280a37a8e8a1a4a2439789
  Additional Information 3:	c91b
  Additional Information 4:	c91bb19bde39930dab9994e275e061d1

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt

**Comment by 5chdn** [comment]
Does it happen without any command line flags?

**Comment by ButteryGuitars** [comment]
Yes, with no flags or config file it still crashes. Let me know if anything else I can provide to help troubleshoot

**Comment by joshua-mir** [comment]
@ButteryGuitars the crash you are sharing appears to be related to kernelbase.dll which is a component of windows, this error seems to occur when the file is corrupted somehow and the solution appears to be to run the [windows system file checker](https://support.microsoft.com/en-us/help/929833/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system) to repair the damage.

**Comment by ButteryGuitars** [comment]
Just an FYI, sfc /scannow did not find any repairs needed. I can replicate this issue on multiple hosts.

**Comment by ButteryGuitars** [comment]
This is resolved in 2.0.8 stable. I think this can safely be closed now

**Comment by 5chdn** [comment]
:tada:

**Comment by joshua-mir** [comment]
@ButteryGuitars is this the issue that has come up again?

**Comment by ButteryGuitars** [comment]
@joshua-mir Yes
